python/pyspark/ml/tuning.py

```diff
@@ -255,18 +257,30 @@ def _fit(self, dataset):
         randCol = self.uid + "_rand"
         df = dataset.select("*", rand(seed).alias(randCol))
         metrics = [0.0] * numModels
+
+        parallelism = self.getParallelism()
+        if parallelism < 1:
+            raise ValueError("parallelism should >= 1.")
```

Comment:
Sorry, I asked for this check but I don't think we really need it.  The ValueError from creating a ThreadPool with processes < 1 is pretty easy to see the problem.  If you want to leave it that's fine too.